keyvi/include/keyvi/index/internal/index_writer_worker.h

```diff
@@ -85,7 +90,11 @@ class IndexWriterWorker final {
     boost::filesystem::path index_toc_file_part_;
     internal::IndexSettings settings_;
     std::list<MergeJob> merge_jobs_;
-    size_t max_concurrent_merges_ = 2;
+    size_t max_concurrent_merges_;
+    size_t max_segments_;
+    size_t compile_key_threshold_;
+    size_t index_refresh_interval_;
```

Comment:
Looks like these members are only read after initialization, so we can have them `const` and initialize in constructor initializer list.